rest/revocation_test.go

```diff
@@ -2257,8 +2257,7 @@ func TestRevocationMessage(t *testing.T) {
 		assert.NoError(t, err)
 
 		// Wait for doc revision to come over
-		_, ok := btcRunner.WaitForBlipRevMessage(btc.id, "doc", version)
-		require.True(t, ok)
+		_ = btcRunner.WaitForBlipRevMessage(btc.id, "doc", version)
```

Comment:
Nitpick: Do we need to have the empty return values on these in this file? Could keep it in line with what you have done in most other places and not have the empty return values.

Reply:
`WaitForBlipRevMessage` does use return value in blip_api_delta_sync_test.go

Reply:
What I meant was you could have `btcRunner.WaitForBlipRevMessage(btc.id, "doc", version)` as opposed to `_ = btcRunner.WaitForBlipRevMessage(btc.id, "doc", version)` 
It doesn't have any impact on tests so i'm good merging, was just a nitpick.